test/bin/setup_local.sh

```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
```

Comment:
Why delete this file?

Reply:
There are quite a few files under `test/bin` that run command sequences. This one called `ci_phase_iso_build` (a composition of other commands by itself) and `start_webserver.sh`.
Note that web server is already started from the `phase` script, so I thought this one is redundant.
Finally, it is not documented on when it should be called.

Reply:
OK, we should document it.

The intent was to provide a single script for a developer to use to build their images and set up to the point of being able to run a single scenario of their choice on a local system.

Reply:
I thought that was `ci_phase_iso_build` for a full build, or just build_images -s